lib/reader.js

```diff
@@ -97,15 +97,8 @@ function Reader (props, currentStat) {
 
   self._path = self.path = path.resolve(props.path)
   if (process.platform === 'win32') {
```

Comment:
We're masochists, but we're not _that_ masochistic. This can go, then. Thanks, Bert!